pages/index.vue

```diff
@@ -17,13 +18,16 @@
 <script>
 export default {
   async asyncData({ $content }) {
-    const { hero, lead, details, cta } = await $content('home').fetch()
+    const { hero, lead, details, cta, projects } = await $content(
+      'home'
+    ).fetch()
```

Comment:
Can we use `$content` in other components? I am thinking about loading this in the `Projects` component.